.gitignore

```diff
@@ -6,3 +6,29 @@
 /node_modules/*
 /yarn.lock
 /yarn-error.log
+
```

Comment:
Nitpick: I think I'd prefer to consider changes to .gitignore in a separate issue/PR.